package.json

```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "test": "ava",
-    "watch": "ava --watch"
+    "test": "npm run lint && ava",
+    "watch": "ava --watch",
+    "lint": "jscs index.js lib test"
```

Comment:
should we add `jscs` and `jshint` as devDependencies and then run them as`./node_modules/bin/jshint`?